### PR TITLE
[WIP] defer attaching inspector until internal code runs

### PR DESCRIPTION
**Pull request by bartlomieju**
Blocked by #5223

### CODE REVIEW

File: cli/worker.rs [hunk 1]
```diff
@@ -128,10 +115,32 @@ impl Worker {
...
       internal_channels,
       external_channels,
-      inspector,
+      inspector: None,
     }
   }
 
+  pub fn attach_inspector(&mut self) {
+    let s = self.state.borrow();
+    let global_state = s.global_state.clone();
```
Comment (piscisaureus) [inline cli/worker.rs:124]: Why does it need to be cloned?

Comment (bartlomieju) [reply cli/worker.rs:124]: @piscisaureus ignore for now; this is a bit more complex problem that I have idea for; I'll try in the morning

Comment (piscisaureus) [reply cli/worker.rs:124]: Took the liberty of removing it myself and pushing it to your branch.

**Comment by bartlomieju** [comment]
Done in #5250

**Pull request closed without merging**